src/multisig/sign/r2.rs

```diff
@@ -88,7 +88,7 @@ impl Executer for R2 {
                 .share_to_party_subshare_ids(peer_keygen_id)?;
 
             valid_signatures.push(SignatureShare {
-                signature,
+                signature_bytes: signature.as_ref().to_der().as_bytes().to_vec(),
```

Comment:
That's a mouthful

Reply:
Indeed.  I added a helper method to `k256_serde`.  The methods in there are a mess, especially for `ProjectivePoint`.  I started cleaning those up, too, but then stopped because it's out of scope and I don't have time. 🙂

Reply:
Nice, thanks!